bc_obps/reporting/service/report_review_changes_service.py

```diff
@@ -292,7 +345,7 @@ def get_report_version_diff_changes(cls, previous: dict, current: dict) -> List[
                 )
 
         # Detect added facilities
-        for fac_name, curr_fac in curr_facilities.items():
+        for fac_name, curr_fac in list(curr_facilities.items()):
```

Comment:
I don't think this does anything

```suggestion
        for fac_name, curr_fac in curr_facilities.items():
```